lib/route.js

```diff
@@ -267,20 +267,9 @@ function buildRouting (options) {
   // HTTP request entry point, the routing has already been executed
   function routeHandler (req, res, params, context) {
     if (closing === true) {
-      const headers = {
-        'Content-Type': 'application/json',
-        'Content-Length': '80'
-      }
-      if (req.httpVersionMajor !== 2) {
-        headers.Connection = 'close'
-      }
-      res.writeHead(503, headers)
-      res.end('{"error":"Service Unavailable","message":"Service Unavailable","statusCode":503}')
       if (req.httpVersionMajor !== 2) {
-        // This is not needed in HTTP/2
-        setImmediate(() => req.destroy())
```

Comment:
Would you mind restoring this but putting it in `reply.res.on('finish')`?

Reply:
done